cmd/use.go

```diff
@@ -144,6 +154,16 @@ func GetDatasetRefString(f Factory, args []string, index int) (string, error) {
 	return refs[0], nil
 }
 
+// GetLinkedFilesysRef returns whether the current directory is linked to a dataset in your repo,
+// and the reference to that dataset.
+func GetLinkedFilesysRef() (bool, string) {
+	data, err := ioutil.ReadFile(QriRefFilename)
+	if err == nil {
+		return true, string(data)
```

Comment:
need to `return strings.TrimSpace(string(data), true`. Newlines in that file are hurting things. Text editors often auto-add newline endings to files these days 🙍‍♂